django/db/backends/base/schema.py

```diff
@@ -433,6 +434,21 @@ def add_field(self, model, field):
         db_params = field.db_parameters(connection=self.connection)
         if db_params['check']:
             definition += " " + self.sql_check_constraint % db_params
+        if field.remote_field and self.connection.features.supports_foreign_keys and field.db_constraint:
+            constraint_suffix = "_fk_%(to_table)s_%(to_column)s"
+            # Add FK constraint inline, if possible
```

Comment:
Trailing dot is missing.

Reply:
If you insist. Again, I was doing my best to be consistent with the surrounding code. None of the other comments for this particular function end with a period.

Reply:
Fixed.